src/logging.c

```diff
@@ -63,3 +63,17 @@ void log_angular_speed(void)
 	LOG_INFO("%f,%f,%f,%d,%d", target_speed, ideal_speed, angular_speed,
 		 pwm_left, pwm_right);
 }
+
+/**
+ * @brief Log sensors distance: side left, side right, front left and front
+ * right.
+ */
+void log_sensors_distance(void)
+{
+	float sl_dist = get_side_left_distance();
+	float sr_dist = get_side_right_distance();
+	float fl_dist = get_front_left_distance();
+	float fr_dist = get_front_right_distance();
+
+	LOG_INFO("%f, %f, %f, %f", sl_dist, sr_dist, fl_dist, fr_dist);
```

Comment:
I would not use spaces after the comma. Although I think it does not have practical implications, it is more standard (use commas to separate values, CSV style, not "comma+space").

Reply:
Good for CSV style, I deleted the spaces.